docs/source/blog/arc-collaboration.md

```diff
@@ -0,0 +1,90 @@
+[Our other examples](https://movement.neuroinformatics.dev/examples/index.html) have also been updated to use these functions where possible.
+
+### Broadcasting Methods
+
+Most of the time during analysis of experimental data, we want to repeat the same action multiple times across different coordinates, or individuals, or time-points.
+The data-structures that `movement` relies on - and the analysis methods it provides - go a long way to making this kind of analysis easy, thanks to array broadcasting.
+However when it comes to writing your own, custom analysis functions, most of the time it is much easier to write a function that takes in a single input, rather than writing an appropriate function that acts on an entire `DataArray`.
+But this approach typically means that you're then stuck writing a `for` loop over the values in your `DataArray`, and on top of that having to copy across the dimension labels or coordinates into the new array you're making as the `for` loop runs.
+
+With this in mind, we have decided to expose the {func}`movement.utils.broadcasting.make_broadcastable` decorator, which can turn functions that take in one-dimensional data (e.g., a single spatial coordinate, or a single time-point) into functions that act across `DataArray`s with appropriate dimensions.
+You can find a new {ref}`example <sphx_glr_examples_broadcasting_your_own_methods.py>` on how to use this decorator on your own functions.
+
+### Regions of Interest (RoIs)
+
+One of the biggest feature introductions to `movement` during this period is {mod}`support for RoIs <movement.roi>`.
+We now support one-dimensional regions of interest (segments, or piecewise-linear structures formed of multiple segments) as well as two-dimensional polygonal regions.
+You can create RoIs using the {class}`movement.roi.LineOfInterest` and {class}`movement.roi.PolygonOfInterest` classes respectively.
+There are more details about how the class is implemented on top of [`shapely`](https://shapely.readthedocs.io/en/stable/) in the {class}`docstring <movement.roi.base.BaseRegionOfInterest>`, and a note about how the interior of 2D shapes is handled.
+
+RoIs support the calculation of certain quantities from experimental data.
+Highlights include;
+
+- Determining if a given point(s) is inside the RoI, {func}`movement.roi.base.BaseRegionOfInterest.contains_point`.
+- Determining the distance from a given point(s) to the closest point on the RoI, {func}`movement.roi.base.BaseRegionOfInterest.compute_distance_to`.
+- Determining the approach vector from a given point(s) to the RoI, {func}`movement.roi.base.BaseRegionOfInterest.compute_approach_vector`.
+- Determining egocentric and allocentric boundary angles, relative to a given RoI. See {func}`movement.roi.base.BaseRegionOfInterest.compute_allocentric_angle_to_nearest_point` and {func}`movement.roi.base.BaseRegionOfInterest.compute_egocentric_angle_to_nearest_point` for more information.
```

Comment:
Because the full function paths are too long here, I would again reach for a syntax like this:

```
{func}`BaseRegionOfInterest.contains_point() <movement.roi.base.BaseRegionOfInterest.contains_point>
```

Reply:
If you decide to run with this, obviously apply it across the bullet points. Also fine if you keep thinks as they are.